src/api/foreground/foreground-setup-embedded-wallet-sdk.ts

```diff
@@ -132,29 +132,4 @@ export function setupEmbeddedWalletSDK(
 
   // @ts-expect-error
   window.arweaveWallet = walletAPI;
-
-  // at the end of the injected script,
-  // we dispatch the wallet loaded event
-  dispatchEvent(new CustomEvent("arweaveWalletLoaded", { detail: {} }));
```

Comment:
For Connect, these are triggered by the SDK after 1) it's instantiated, 2) the wallet API is injected into the page and 3) the authentication process completes and the wallet is ready to be used.